generators/profilegen/src/profile_template_files.rb

```diff
@@ -7,7 +7,15 @@
 # in the generation of profile SDKs
 #
 module ProfileTemplateFiles
-  MODULE_TEMPLATE = './resources/templates/module_template.template'
-  CLIENT_TEMPLATE = './resources/templates/client_template.template'
-  MODULE_DEFINITION_TEMPLATE = './resources/templates/module_definition_template.template'
+  module Rollup
```

Comment:
are these constants only used in the profile_template.rb file? if so, should we define them directly there?